### PR TITLE
Allow using videos as screenshots

### PR DESCRIPTION
**Pull request by kidonng**
Both a bug fix and an enhancement.

- Bug fix: the readme parser doesn't include `jpg` in the screenshot regex, resulting in many screenshots being unavailable in the options window.
- Enhancement: [`.mp4` and `.mov` videos are now supported](https://github.blog/2021-05-13-video-uploads-available-github/) in the options window. In fact, `select-notifications` already uses a video as the screenshot.

```
$ rg --only-matching --replace '$1' 'user-images.+\.(\w+)\)$' readme.md | sort -u
gif
jpg
mp4
png
```

## Test URLs

chrome://extensions/?options=balgliakcbamepammkbcjdepogihccmj

## Screenshot

image

### CODE REVIEW

File: source/options.tsx [hunk 1]
```diff
@@ -165,8 +167,8 @@ function summaryHandler(event: delegate.Event<MouseEvent>): void {
 	const toggle = feature.querySelector('input.screenshot-toggle')!;
 	toggle.checked = !toggle.checked;
 
-	// Lazy-load image
-	const screenshot = feature.querySelector('img.screenshot')!;
+	// Lazy-load image/video
+	const screenshot = feature.querySelector<HTMLImageElement | HTMLVideoElement>('.screenshot')!;
 	screenshot.src = screenshot.dataset.src!;
```
Comment (fregante) [inline source/options.tsx:170]: Do videos work in image tags in every browser?

Comment (kidonng) [reply source/options.tsx:170]: Maybe you forgot to read the changes above...

Comment (fregante) [reply source/options.tsx:170]: Ooh, iyaa

**Review by fregante — APPROVED** [review]

**Comment by darkred** [comment]
@kidonng the two image links ([1](https://github.com/refined-github/refined-github/pull/5271/commits/e5bf2a19aaa78c38214ca77a811773699cb9c54f#diff-5a831ea67cf5cf8703b0de46901ab25bd191f56b320053be9332d9a3b0d01d15L232), [2](https://github.com/refined-github/refined-github/commit/e5bf2a19aaa78c38214ca77a811773699cb9c54f#diff-5a831ea67cf5cf8703b0de46901ab25bd191f56b320053be9332d9a3b0d01d15L318)) in which you changed the file extensions to lowercase (`.PNG` --> `.png`)
have become broken. I believe they should remain in uppercase.

**Comment by cheap-glitch** [comment]
> I believe they should remain in uppercase, as they were.

Or we could just re-upload the screenshots, for consistency and ease of parsing.

**Comment by darkred** [comment]
Yes, that would be the best.

**Comment by kidonng** [comment]
Adding a note here: `.mp4` videos are preferred because their URLs can opened and played directly in the browser. `.mov`'s URL only downloads it.

**Pull request merged**